rqd/rqd/rqcore.py

```diff
@@ -122,13 +122,22 @@ def __createEnvVariables(self):
         if 'GPU_LIST' in self.runFrame.attributes:
             self.frameEnv['CUE_GPU_CORES'] = self.runFrame.attributes['GPU_LIST']
 
+        # Add host environment variables
+        if rqd.rqconstants.RQD_USE_HOST_ENV_VARS:
+            for key, value in os.environ.items():
+                if "PATH" in key and key in self.frameEnv:
+                    self.frameEnv[key] += os.pathsep + value
+
+                self.frameEnv[key] = value
```

Comment:
Change the logic to only catch `PYTHONPATH`

Reply:
This could be conflicting with the feature introduced in this PR (still in review) :
https://github.com/AcademySoftwareFoundation/OpenCue/pull/1270

Reply:
In fact the feature you propose in this PR might be a good solution for the use case I describe [here](https://github.com/AcademySoftwareFoundation/OpenCue/pull/1324#discussion_r1387785285)